apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts

```diff
@@ -557,16 +557,14 @@ describe('getAppealInfo', () => {
     expect(appealInfo).toEqual({
       canBeAppealed: true,
       hasBeenAppealed: false,
-      appealDeadline: new Date(
-        new Date(rulingDate).setDate(new Date(rulingDate).getDate() + 3),
-      ).toISOString(),
+      appealDeadline: '2022-06-18T19:50:08.033Z',
```

Comment:
Changing this to static dates to avoid adding logic when setting expected values

Reply:
Nice!